scripts/ci/config.json

```diff
@@ -1,12 +1,16 @@
 {
```

Comment:
provide both "dev" and regular verisons. "dev" are build on evg/locally, regular is final destination for image.